src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -118,6 +118,10 @@ private PipelineOptions buildPipelineOptions() {
             options.setProject(projectID);
             options.setStagingLocation(stagingLocation);
             options.setRunner(this.runnerType.runner);
+            // n1-standard-4 is 4x the RAM and 4x the CPUs as the default machine, at only 4x the price.
+            options.setWorkerMachineType("n1-standard-4");
```

Comment:
Expose machine type as a command-line arg in `DataflowCommandLineProgram` like the other pipeline options, with `"n1-standard-4"` as the default value.